core/primitives/src/views.rs

```diff
@@ -378,6 +378,14 @@ impl From<Tip> for BlockStatusView {
     }
 }
 
+#[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
+#[derive(Serialize, Deserialize, Debug)]
+pub struct EpochInfoView {
+    pub epoch_id: CryptoHash,
+    pub start_time: String,
```

Comment:
Can we use the actual Instant here instead of String. You can get around the DeepSizeFeature requirement by using this https://github.com/near/nearcore/pull/6467/files#diff-c0cb71be9c57de08ac1ac5752b7af7a55f7ff6226435f786d8199baecc0bb4f4R58

Reply:
I think of this struct as a ready-to-display payload for javascript. The purpose here is to provide a value that frontend can just print without having to do any conversion. In that regard, maybe a string is the easiest.

Reply:
I think what the javascript is getting is a json object, which is serialized by this Serialize trait here, so you don't have to worry about converting the inner fields into string. Letting the serialization be handled automatically help with the consistency of your output.